dannce/engine/generator.py

```diff
@@ -1687,6 +2205,16 @@ def visualize(self, original, augmented):
         input("Press Enter to continue...")
 
     def random_continuous_rotation(self, X, y_3d, max_delta=5):
+        """Summary
+
+        Args:
+            X (TYPE): Description
+            y_3d (TYPE): Description
+            max_delta (int, optional): Description
```

Comment:
`max_delta (int, optional): maximum range for rotation angle.`

Reply:
```suggestion
            max_delta (int, optional): maximum range for rotation angle.
```